ext/arm/core.lb

```diff
@@ -27,7 +27,10 @@ def build(env):
     env.copy("cmsis/CMSIS/Core/Include/cmsis_version.h", "cmsis_version.h")
     env.copy("cmsis/CMSIS/Core/Include/cmsis_compiler.h", "cmsis_compiler.h")
     env.copy("cmsis/CMSIS/Core/Include/cmsis_gcc.h", "cmsis_gcc.h")
-    if "m0" not in core:
+    if core != "0": # 0+ has MPU support though!
         env.copy("cmsis/CMSIS/Core/Include/mpu_armv7.h", "mpu_armv7.h")
+    print(core)
```

Comment:
Do we need to print the core version?

Reply:
I'm getting old, I need to be reminded what I'm working with every build!1!!

Reply:
After merging the DMA PR, I'll push a commit to remove this.